recipe/bld.bat

```diff
@@ -1,46 +1,37 @@
-@echo on
+copy "%RECIPE_DIR%\bld_win.sh" .
```

Comment:
Is this copy necessary? Could you not just pass the directory into the bash call later?

Reply:
Thanks Marco! I got rid of that.